lms/templates/courseware/courseware.html

```diff
@@ -89,31 +92,10 @@
   % endif
 
   <script type="text/javascript">
-    var $$course_id = "${course.id | escapejs}";
-
-    $(function(){
-        $(".ui-accordion-header a, .ui-accordion-content .subtitle-name").each(function() {
```

Comment:
Thana @andy-armstrong. I thought it was probably used, but couldn't get this to do anything because when trying `$(".ui-accordion-header a, .ui-accordion-content .subtitle-name")`, this always came up empty for me. Is this user error on my part?